src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepository.scala

```diff
@@ -38,3 +62,9 @@ class FileMetadataRepository(db: Database)(implicit val executionContext: Execut
     db.run(query.result)
   }
 }
+
+object FileMetadataRepository {
+
+  case class FileMetadataRowWithName(propertyName: String, metadataid: UUID, fileid: UUID, value: String, datetime: Timestamp, userid: UUID)
```

Comment:
I'm guessing you and Tom have already discussed this PR, since this class looks very similar to the consignment one in #107? I accidentally reviewed that one first even though it's newer because it's listed first by the PR monitor.

What do you think about [this comment](https://github.com/nationalarchives/tdr-consignment-api/pull/107#discussion_r552056164), about whether this class should go in the Repository or the Service and whether it needs all the fields?

Reply:
The reason this is in the repository is that the `propertyName` field and the all the other fields come from two different tables. So the only way I can think of of constructing this object in the service is to return a tuple like `("propertyName", "metadataValuesRow")` from the repository which never feels as clean as returning an object. 

Also, there are more than likely going to be other things calling these repository methods from different services and if the logic to create the object is in there, it will be duplicated.